packages/devtools-launchpad/webpack.config.js

```diff
@@ -48,6 +48,11 @@ module.exports = (webpackConfig, envConfig) => {
     loader: "svg-inline"
   });
 
+  // Add resolveLoader for ./node_modules to fix issues when synlinked.
+  webpackConfig.resolveLoader = webpackConfig.resolveLoader || {};
+  webpackConfig.resolveLoader.root = webpackConfig.resolveLoader.root || [];
+  webpackConfig.resolveLoader.root.push(path.resolve("./node_modules"));
```

Comment:
This fixes the 'cannot find module babel' errors. Added it after looking at the conversation at https://github.com/webpack/webpack/issues/784